internal/backend/remote-state/oss/backend.go

```diff
@@ -379,12 +379,13 @@ func (b *Backend) configure(ctx context.Context) error {
 	if endpoint == "" {
 		endpointsResponse, err := b.getOSSEndpointByRegion(accessKey, secretKey, securityToken, region)
 		if err != nil {
-			return err
-		}
-		for _, endpointItem := range endpointsResponse.Endpoints.Endpoint {
-			if endpointItem.Type == "openAPI" {
-				endpoint = endpointItem.Endpoint
-				break
+			log.Printf("[WARN] getting oss endpoint failed and using oss-%s.aliyuncs.com instead. Error: %#v.", region, err)
+		}else {
```

Comment:
I think this is the line that `gofmt` is failing with.

Reply:
Thanks for your review and feedback and I have fix it. Please have a check again.